Description: A patch for AUTHORS before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/AUTHORS
+++ autotrace-0.31.1/AUTHORS
@@ -1,2 +1,2 @@
 Martin Weber
-martweb@gmx.net
\ No newline at end of file
+martweb@gmx.net
